src/components/grid-aware/TwoParagraphBlock/TwoParagraphBlock.module.css

```diff
@@ -0,0 +1,143 @@
+/* Background bleed */
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  grid-template-rows: auto 0;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+.bleedBackground {
+  background: var(--color-gray-300);
+  grid-column: 1 / span 2;
+  grid-row: 1 / -1;
+}
+
+.bleedMainContent {
+  grid-column: 2;
+  grid-row: 1;
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    grid-template-columns: 40px 1fr;
+  }
+
+  .bleedBackground {
+    grid-column: 1 / span 3;
+  }
+}
+
+
+/* Main grid */
+
+.gridParent {
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+  grid-template-rows: auto 590px;
+  padding-top: 122px;
+  row-gap: 84px;
+}
+
+.gridParent > * {
+  min-width: 0;
```

Comment:
No, the reason this is here is because the default min-width is `auto` (and `auto` means however wide it needs to be to fit its content), which means that a child can stretch out its parent. This is most noticeable on images, since a large image can then take up more space than allocated to it unless if we have this property set.